packages/icons/tests/chromatic/Icons.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import type { ContextValue } from "react-aria-components";
+import type { ResponsiveProp, ColorValue } from "@hopper-ui/styled-system";
+import * as IconLibrary from "@hopper-ui/icons";
+
+import "./icons.stories.css";
+
+interface IconLibraryType {
+    [key: string]: React.ComponentType<ContextValue<Omit<IconLibrary.IconProps, "src16" | "src24" | "src32" >, SVGSVGElement>>;
+}
+
+interface ListProps {
+    icons: string[];
+    size?: ResponsiveProp<"sm" | "md" | "lg">;
+    color?: ResponsiveProp<ColorValue>;
+}
+
+const List = ({ icons, size, color }: ListProps) => {
+    const listItems = icons.map(name => {
+        const Component = (IconLibrary as unknown as IconLibraryType)[name];
+
+        return (
+            <li className="icons-list__item" key={name} >
+                <Component size={size} color={color} />
+            </li>);
+    });
+
+    return <ul className="icons-list">{listItems}</ul>;
+};
+
+export default {
```

Comment:
The way to type stories has changed twice since this synthax. 

The "as Meta<typeof List>;" isn't ideal, since it's basically a "type casting"  instead of typing the element. TypeScript released the `satisfies` keyword, so it has been updated with it

```
    title: "Icons/Icons"
} satisfies Meta<typeof List>;
```
This way, it doesn't cast, but ensures that the type satisfies the interface

But with the latest storybook update, StoryObj now takes the meta as it's type parameter, allowing better typing of props. So the new synthax is 

```
const meta: Meta<typeof List> = {
    component: List,
    title: "Icons/Icons"
};

export default meta;

type Story = StoryObj<typeof meta>;
```

Reply:
alright thanks. I'm surprised you didn't put export default const meta: Meta since you usually like one liners if possible.

Reply:
```
export default const meta
```
This doesn't seem like a valid synthax sadly

Reply:
really? so export const works but not export default const.. hm ok then. That explains it. Guess I never really tested that.